starboard/raspi/shared/BUILD.gn

```diff
@@ -378,8 +378,13 @@ static_library("starboard_base_symbolize") {
     "//third_party/symbolize/demangle.cc",
     "//third_party/symbolize/symbolize.cc",
   ]
-
-  public_deps = [ "//starboard/elf_loader:evergreen_info" ]
+  deps = [
```

Comment:
if things were building without this , why was this added now ?

Reply:
Because the symbolize.cc uses  a few starboard headers, and previously those were pulled in transitively via elf_loader.